docs/webhooks/11-Webhook-IPs.md

```diff
@@ -0,0 +1,65 @@
+---
+tags: [webhooks]
+---
+> Notice: This safelist of IP addresses will take effect April 28th, 2022 - 14:00 UTC
+
+# Webhook Notifications
+
+If you are consuming Generic Webhooks from PagerDuty, use these IPs to make sure make sure they originate from a known PagerDuty Safelist IP.
+
+
+The full list of IP addresses that our webhook notifications may come from is:
+
+### US Customers:
```

Comment:
In other places we refer to these as "Service Regions":

* https://developer.pagerduty.com/docs/ZG9jOjE4NzkyNjM1-public-certificates
* https://support.pagerduty.com/docs/service-regions